libexec/scoop-shim.ps1

```diff
@@ -0,0 +1,211 @@
+# Usage: scoop shim [<subcommand>] <shim_names> [<command_path> [<args>...]]
+# Summary: Manipulate Scoop shims
+# Help: Manipulate Scoop shims: add, remove/rm, list, info, alter, etc.
+#
+# To add a costom shim, use the 'add' subcommand:
+#
+#     scoop shim add <shim_name> <command_path> [<args>...]
+#
+# To remove a shim, use the 'remove' or 'rm' subcommand (CAUTION: this could remove shims added by app manifest):
+#
+#     scoop shim remove <shim_names>
+#
+# To list all shims or a matching shim, use the 'list' subcommand:
+#
+#     scoop shim list [<shim_name>]
+#
+# To show a shim's information, use the 'info' subcommand:
+#
+#     scoop shim info <shim_name>
+#
+# To alternate a shim's target source, use the 'alter' subcommand:
+#
+#     scoop shim alter <shim_name>
+#
+# Options:
+#   -g, --global       Add/Remove/Info/Alter global shim(s)
+
+param($SubCommand, $ShimName)
+
+. "$PSScriptRoot\..\lib\core.ps1"
```

Comment:
```suggestion
```

`core` is not required because it is imported already in https://github.com/ScoopInstaller/Scoop/blob/21c5c6e775f51ea84eafd436b55078d43490a7e6/bin/scoop.ps1#L6